### ISSUE TITLE
Updating the deployment failed due to busy target

### ISSUE THREAD

**Issue by hfrenzel** [cloudfoundry-community/containers-boshrelease#2]
Trying with anchore.yaml, the update of the deployment failed, as the volume can't be unmounted:

```
Task 704520 | 13:40:10 | Updating instance docker: docker/7b751457-a2e7-45d1-858c-db04f418d94b (0) (canary) (00:11:17)
L Error: Action Failed get_task: Task 1e93d884-9376-4d10-6bcb-e34c6454933f result: Unmounting persistent disk: Running command: 'umount /dev/sdd1', stdout: '', stderr: 'umount: /var/vcap/store/docker: target is busy
```

**Comment by jhunt** [comment]
Interesting.  I shall have to take this for a spin in the lab.

**Comment by aegershman** [comment]
If it means anything, I'm seeing the same thing. You can see this in action by trying the `anchore.yaml` and performing a `recreate` on the deployment. I'm wondering if docker is still marking the volumes being mounted as 'in-use' and not letting go.

I also tested this with something different & mounted files created with the `files` job property into containers with `volumes`, and saw something very similar. I can post the full deployment if desired (or in another issue; don't mean to hijack this)

```yml

...
files:
  default.yml: |
    xyzabc
recipe:
  services:
     splunk:
       volumes:
        - /var/vcap/jobs/docker/etc/running/default.yml:/tmp/defaults/default.yml
```

then when performing a `recreate` (or stemcell bump, etc.), the task fails with an error message similar to hfrenzel:

```log
ask 20733655 | 00:39:21 | Updating instance docker: docker/efc96e44-f0ad-42a9-b592-ea238b134774 (0) (canary) (00:10:17)
                     L Error: Action Failed get_task: Task fce63add-68ec-4aa8-5bb5-b6374d2fee7b result: Unmounting persistent disk: Running command: 'umount /dev/nvme2n1p1', stdout: '', stderr: 'umount: /var/vcap/store/docker: target is busy
        (In some cases useful info about processes that
         use the device is found by lsof(8) or fuser(1).)
': exit status 32
Task 20733655 | 00:49:38 | Error: Action Failed get_task: Task fce63add-68ec-4aa8-5bb5-b6374d2fee7b result: Unmounting persistent disk: Running command: 'umount /dev/nvme2n1p1', stdout: '', stderr: 'umount: /var/vcap/store/docker: target is busy
        (In some cases useful info about processes that
         use the device is found by lsof(8) or fuser(1).)
': exit status 32
```

**Comment by jhunt** [comment]
I have been able to reproduce this by deploying the `manifests/anchore.yml` example and then attempting a `bosh recreate`.  My guess is there is something wrong with the `compose stop` teardown logic.

**Comment by jhunt** [comment]
This should be fixed in the 1.1.0 release: https://github.com/cloudfoundry-community/containers-boshrelease/releases/tag/v1.1.0

🍻

**Comment by aegershman** [comment]
Did a quick `--recreate` test and can confirm it's releasing the volumes as expected 👌 thank you
